src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/color_widget.html.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
```

Comment:
no other file in this directory has the license header, we should removed

Reply:
It was suggested by fabbot, but I'll revert the changes ok